### PR TITLE
Automatically publish to NPM

### PR DESCRIPTION
**Pull request by luisamiranda**

### CODE REVIEW

**Comment by luisamiranda** [comment]
Please excuse and SQUASH the 136 commits it took me to get this right. Yeesh.

**Comment by elizabethhalper** [comment]
hey! could you rebase from master? I think there are a few changes that you're missing

**Comment by luisamiranda** [comment]
@elizabethhalper - I just rebased again. It SHOULD be up to date with master.

File: azure-pipelines/linux/continuous-build-linux.yml [hunk 1]
```diff
@@ -27,11 +27,16 @@ steps:
...
     # https://docs.codecov.io/docs/about-the-codecov-bash-uploader
     bash <(curl -s https://codecov.io/bash) -t $(CODECOV_TOKEN)
   displayName: 'Upload coverage report'
+
+# - bash: |
+#     npm run build-lib
+#     npm install -g npm-cli-login
+#     npm-cli-login -u $(NPM_USER) -p $(NPM_PASS) -e $(NPM_EMAIL)
+#     npm version patch
+#     npm publish
```
Comment (elizabethhalper) [inline azure-pipelines/linux/continuous-build-linux.yml]: Delete commented out code

**Pull request merged**